File: Leetcode75/392. Is Subsequence.java
class Solution {
    public boolean isSubsequence(String s, String t) {
        int l = 0; 
        if(s.length() > t.length()) return false;
        if(s.length() == 0) return true;
        for(int i = 0; i < t.length(); i++){
            if(t.charAt(i) == s.charAt(l)){
                l++;
                if(l == s.length()) return true;
            }
        }
        return l==s.length()?true:false;
    }
}

File: PastYears/Google/392_Is_Subsequence.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
        int left = 0;
        
        for(int i = 0; i < t.length(); i++){
            if(t[i] == s[left]) left++;
            if(left == s.length()) return true;
        }
        /*In case t.length() < s.length()*/
        return left == s.length()?true:false;
    }
};
